src/ports/postgres/modules/deep_learning/test/madlib_keras_evaluate.sql_in

```diff
@@ -59,3 +61,44 @@ ALTER TABLE keras_saved_out DROP COLUMN model_arch;
 SELECT assert(trap_error($TRAP$
 	SELECT madlib_keras_evaluate('keras_saved_out', 'cifar_10_sample_val', 'evaluate_out');
 	$TRAP$) = 1, 'Should error out if model_arch column is missing from model_table');
+
+
+
+m4_changequote(`<!', `!>')
+m4_ifdef(<!__POSTGRESQL__!>, <!!>, <!
+
+DROP TABLE IF EXISTS mst_table, mst_table_summary;
+SELECT load_model_selection_table(
+    'model_arch',
+    'mst_table',
+    ARRAY[1],
+    ARRAY[
+        $$loss='categorical_crossentropy',optimizer='Adam(lr=0.1)',metrics=['accuracy']$$,
+        $$loss='categorical_crossentropy', optimizer='Adam(lr=0.01)',metrics=['accuracy']$$,
+        $$loss='categorical_crossentropy',optimizer='Adam(lr=0.001)',metrics=['accuracy']$$
+    ],
+    ARRAY[
+        $$batch_size=5,epochs=1$$,
+        $$batch_size=10,epochs=1$$
+    ]
+);
+
+DROP TABLE if exists cifar_10_multiple_model, cifar_10_multiple_model_summary,
+                     cifar_10_multiple_model_info;
+SELECT setseed(0);
+SELECT madlib_keras_fit_multiple_model(
+    'cifar_10_sample_batched',
+    'cifar_10_multiple_model',
+    'mst_table',
+    6,
+    0
+);
+
+DROP TABLE IF EXISTS evaluate_out;
+SELECT madlib_keras_evaluate('cifar_10_multiple_model', 'cifar_10_sample_val', 'evaluate_out', 0, 1);
+SELECT assert(loss IS NOT NULL AND
```

Comment:
instead of checking for NULL, do you think it makes sense to check for `>= 0` ?

Reply:
Should we change the NULL check for all of the tests?

Reply:
yeah sure while we are at it